pages/approval.tsx

```diff
@@ -145,24 +159,34 @@ const Approval: NextPage = () => {
           ))}
         </Grid>
         <div style={{ padding: 20 }}>
-          <Link
-            href={{
-              pathname: '/assessment',
-              query: { task: taskNumber },
-            }}
-            passHref
-          >
-            <Button variant="contained">Tilbake</Button>
-          </Link>
-          <Link href="/task" passHref>
-            <Button
-              style={{ marginLeft: 10 }}
-              variant="contained"
-              onClick={() => saveAssessments(assessments, key)}
+          {checkInconsistentScores(assessments) == true ? (
```

Comment:
I think it should be checked on if there are un-set scores.
E.g. something ish like this
```
function checkIfAllScoresAreSet(assessments) {
  return !assessments.some((e) => e.score === ''
```

Because u can unset scores, and then this in possible:
<img width="1019" alt="image" src="https://user-images.githubusercontent.com/31897129/163021427-d1d3c82f-41a0-4875-8a6a-4e0f8e5f067d.png">

Reply:
I fixed it in #125. Bad branch name, I know... I was supposed to only fix one thing but did it there once I started cause I got so caught up 🤓

Reply:
Oh, okay! 👏  My bad!